pyproject.toml

```diff
@@ -16,7 +16,9 @@ dependencies = [
     "meds == 0.1.3",
     "typing_extensions >= 4.0",
     "datasets >= 2.0.0",
-    "tqdm >= 4.6.0"
+    "tqdm >= 4.6.0",
+    "duckdb >= 0.10.1",
```

Comment:
Can we remove duckdb here from the dependency list and add it as an optional dependency? Similar to how I have meds_etl_cpp as an optional dependency in my PR?

Reply:
done